.github/workflows/trigger-nightly.yml

```diff
@@ -0,0 +1,27 @@
+name: Trigger nightly builds
+
+on:
+  schedule:
+    # every night at 4:30AM
+    - cron: 30 4 * * *
```

Comment:
What is the schedule for PyTorch core? We need to make sure to only run this after todays core nightly has been build. Otherwise we'll publish "todays" torchvision nightly based on "yesterdays" core nightly.

Reply:
PyTorch core starts at 0:00 we give some time for it to finish before kicking this cron. This is the same as what we have via pytorch.warm package
Also note: This is the first step - migrating everything to GHA and deprecating pytorch.warm package. 
Next step is building everything core and domains from test-infra and sequencing builds core and then domains, this will take care of the use case you described.